<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demoQtGUIClass</class>
 <widget class="QMainWindow" name="demoQtGUIClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>demoQtGUI</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="componentLineEdit_1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件1</string>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="calculateLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>其他元件坐标：</string>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>425</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="xyLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>查看坐标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="movePushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>移动</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>机器坐标</string>
    </property>
   </widget>
   <widget class="QLabel" name="readFileLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>读取文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="readFileLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>510</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="readFilePushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>读取文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="rowNumLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="displayTextPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>展示文本</string>
    </property>
   </widget>
   <widget class="QLabel" name="originalImageLabel1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>571</width>
      <height>311</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="originalImageLabel2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>571</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cameraLabel1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>相机1</string>
    </property>
   </widget>
   <widget class="QLabel" name="cameraLabel2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>相机2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openCameraPushButton1">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>350</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeCameraPushButton1">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>350</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭相机1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takePhotoPushButton1">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>350</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newWindowPushButton1">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>350</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机1图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="textRowNumberLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>文件行数</string>
    </property>
   </widget>
   <widget class="QLabel" name="machineCoordinateLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="newWindowPushButton2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>720</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机2图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openCameraPushButton2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>720</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeCameraPushButton2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>720</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭相机2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="takePhotoPushButton2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>720</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openSerialPushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>87</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="serialOutputTextBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoPickPushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>自动拾取</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="homographyLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Homography</string>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_8">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件8</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit_9">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_9">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel7">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_9">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件6</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="componentLineEdit_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="componentLabel_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>元件5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xyPushButton_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>物理坐标</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileCoordinateLabel5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="positionPushButton9">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>位置9</string>
    </property>
   </widget>
   <widget class="QLabel" name="savePhotoLabel1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="savePhotoLabel2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>390</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="pointNumberBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="pointNumberLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计算单应矩阵所需点数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calcAllComponentsPushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>全部计算</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1306</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="demoQtGUI.qrc"/>
 </resources>
 <connections/>
</ui>
